components/form/test/form-helper.test.js

```diff
@@ -142,7 +142,7 @@ describe('form-helper', () => {
 
 		const implicitLabelWithTooltipFixture = html`
 			<label>Do you like peas?
-				<input id='target' type="checkbox" name="peas">
+				<input id='target' type="checkbox" name="peas" style="position: relative;">
```

Comment:
The usages in test files like this don't _need_ to have this added, but I was just going through and silencing all the new warnings.